doc/flame/rendering/text.md

```diff
@@ -93,25 +93,32 @@ class MyGame extends FlameGame {
 text inside a bounding box, creating line breaks according to the provided box size.
 
 You can decide if the box should grow as the text is written or if it should be static by the
-`growingBox` variable in the `TextBoxConfig`.
+`growingBox` variable in the `TextBoxConfig`. A static box could either have a fixed size (setting
+the `size` property of the `TextBoxComponent`), or to automatically shrink to fit the text content.
+
+In addition, the `align` property allows you to control the the horizontal and vertical alignment
+of the text content. For example, setting `align` to `Anchor.center` will center the text within
+its bounding box both vertically and horizontally.
 
 If you want to change the margins of the box use the `margins` variable in the `TextBoxConfig`.
 
 Example usage:
 
 ```dart
 class MyTextBox extends TextBoxComponent {
-  MyTextBox(String text) : super(text: text, textRenderer: tiny, boxConfig: TextBoxConfig(timePerChar: 0.05));
+  MyTextBox(String text)
+    : super(text: text, textRenderer: tiny, boxConfig: TextBoxConfig(timePerChar: 0.05));
 
   @override
-  void drawBackground(Canvas c) {
+  void render(Canvas canvas) {
     Rect rect = Rect.fromLTWH(0, 0, width, height);
-    c.drawRect(rect, Paint()..color = Color(0xFFFF00FF));
-    c.drawRect(
+    canvas.drawRect(rect, Paint()..color = Color(0xFFFF00FF));
```

Comment:
We shouldn't encourage our users to create `Paint` objects in the render loop (even though it was done before this PR).

Reply:
fixed